coalib/testing/LocalBearTestHelper.py

```diff
@@ -84,6 +84,7 @@ def check_results(self,
                       local_bear,
                       lines,
                       results,
+                      settings={},
```

Comment:
could you please put the parameter at the end of the parameters list? then this doesn't break current tests on coala-bears which rely on the position of each argument^^